app/controllers/board_sections_controller.rb

```diff
@@ -86,6 +87,19 @@ def require_permission
     end
   end
 
+  def og_data
+    data = {
+      url: board_section_url(@board_section),
+      title: "#{@board_section.board.name} » #{@board_section.name}",
```

Comment:
To keep consistent with the style used for posts, should this instead be in the format `Board section · Board` (instead of `Board » Board section`)? Same goes for some of the other things here. It might be worth asking #constellation on it.

Reply:
Yeah I wanted to consult #constellation on a lot of the formatting tbh, and I figured it'd be easier to do with samples

Reply:
That's totally fair.